type-definitions/tests/immutable-flow.js

```diff
@@ -5,7 +5,7 @@
 // Some tests look like they are repeated in order to avoid false positives.
 // Flow might not complain about an instance of (what it thinks is) T to be assigned to T<K, V>
 
-import Immutable, {
```

Comment:
It tested valid imports. I know it might be confusing though. I should've put a comment there.